From 8ed773749fd59ff4036ded5ad106de027f92cefe Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Thu, 9 Mar 2017 06:12:04 +0100
Subject: [PATCH 16/17] spice: remove spice-experimental.h include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <ed3cc22a1061a82e508c44c2c4b045997185c0e1.1489039263.git.mrezanin@redhat.com>
Patchwork-id: 74258
O-Subject: [RHEL-7.4 qemu-kvm PATCH 1/2] spice: remove spice-experimental.h include
Bugzilla: 1430606
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Marc-André Lureau <mlureau@redhat.com>
RH-Acked-by: David Hildenbrand <david@redhat.com>

From: Marc-André Lureau <marcandre.lureau@gmail.com>

Nothing seems to be using functions from spice-experimental.h (better
that way). Let's remove its inclusion.

Signed-off-by: Marc-André Lureau <marcandre.lureau@gmail.com>
Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
(cherry picked from commit e0883e2de0ef36f254acc274e80ddeac13a2a8f6)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 spice-qemu-char.c | 1 -
 ui/spice-core.c   | 1 -
 2 files changed, 2 deletions(-)

diff --git a/spice-qemu-char.c b/spice-qemu-char.c
index 6d147a7..cd51f3c 100644
--- a/spice-qemu-char.c
+++ b/spice-qemu-char.c
@@ -3,7 +3,6 @@
 #include "ui/qemu-spice.h"
 #include "sysemu/char.h"
 #include <spice.h>
-#include <spice-experimental.h>
 #include <spice/protocol.h>
 
 #include "qemu/osdep.h"
diff --git a/ui/spice-core.c b/ui/spice-core.c
index 8d6e726..0585267 100644
--- a/ui/spice-core.c
+++ b/ui/spice-core.c
@@ -16,7 +16,6 @@
  */
 
 #include <spice.h>
-#include <spice-experimental.h>
 
 #include <netdb.h>
 #include "sysemu/sysemu.h"
-- 
1.8.3.1

